exercises/protein-translation/example.cpp

```diff
@@ -0,0 +1,29 @@
+#include "protein_translation.h"
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <map>
+
+using namespace std;
+
+vector<string> protein_translation::proteins(string rna) {
+
+    map<string, string> dic;
```

Comment:
Nit: Should probably use an `unordered_map` here. It's more explicit about the fact that you don't need them to be in order.

Even better would be a `static const unordered_map`.